tests/collections/asr/decoding/test_cuda_graph_rnnt_greedy_decoding.py

```diff
@@ -11,19 +11,38 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import copy
 import glob
-import tempfile
 
 import jiwer
 import pytest
 import torch
-from omegaconf import OmegaConf, open_dict
+from omegaconf import open_dict
 
 from nemo.collections.asr.models import ASRModel
 from nemo.core.utils.cuda_python_utils import skip_cuda_python_test_if_cuda_graphs_conditional_nodes_not_supported
 
 
+@pytest.fixture(scope="module")
```

Comment:
Shouldnt this be with a marker for `with_download`

Reply:
Pytest marks (`with_downloads`) are useful only for tests, not fixtures. All the tests that use these fixtures are marked with `with_downloads` tag.